src/types/pathways-model.d.ts

```diff
@@ -37,7 +37,7 @@ declare module 'pathways-model' {
   export interface ActionNode extends PathwayNode {
     cql: string;
     elm?: ElmLibrary;
-    action: Action[];
+    action: Action;
```

Comment:
I get why we want this to be required, but since there are currently some examples of places where it's functionally null, just an empty object, it might make more sense just to have it be optional. The `if (action !== ({} as Action))` that I see in a few places is just a weirder looking null check.

Reply:
So this was a little bit more complex than just making it optional since we use `action` to determine if the node is an action node or not. Luckily `null !== undefined` in JS so I made it `Action | null`. There was a lot which needed to be updated with this but it looks like its all working well